cli/crowdin.ts

```diff
@@ -56,26 +92,61 @@ export function internalUploadTargetTranslationsAsync(uploadDocs: boolean) {
                     .then(() => fs.existsSync("built/sim-strings.json") ? execCrowdinAsync("upload", "built/sim-strings.json", crowdinDir) : Promise.resolve())
                     .then(() => uploadBundledTranslationsAsync(crowdinDir, cred.branch, cred.prj, cred.key))
                     .then(() => {
-                        if (uploadDocs) {
+                        if (uploadDocs || pxt.crowdin.testMode) {
```

Comment:
why are you overloading here?

Reply:
Temporary (will add a todo to remove before un-drafting this) - just forcing it so I can see the results of the changes without adding the flags to every other repo